packages/react-server/src/ReactFizzServer.js

```diff
@@ -410,11 +417,40 @@ function popComponentStackInDEV(task: Task): void {
   }
 }
 
-function logRecoverableError(request: Request, error: mixed): void {
+// stash the component stack of an unwinding error until it is processed
+let lastBoundaryErrorComponentStackDev: ?string = null;
+
+function captureBoundaryErrorDetailsDev(
+  boundary: SuspenseBoundary,
+  error: mixed,
+) {
+  let errorMessage;
+  if (typeof error === 'string') {
+    errorMessage = error;
+  } else if (error && typeof error.message === 'string') {
+    errorMessage = error.message;
+  } else {
+    // eslint-disable-next-line react-internal/safe-string-coercion
+    errorMessage = String(error);
+  }
+
+  let errorComponentStack = lastBoundaryErrorComponentStackDev;
+  if (errorComponentStack === null) {
+    errorComponentStack =
+      '[A component stack was not captured at the point of origin of the original error. ' +
+      'This Component stack is from the boundary handling the error.]\n' +
```

Comment:
The component stacks are parsed. The format is the same as the native stacks also any error parser can do it. This introduces a new part to that string that can break parsers.

It’s not only human readable.

Reply:
Ah... good to know. i'll remove. I suppose it's not that useful and should be impossible today